### ISSUE TITLE
Cannot create PR here from my gitflow-avh fork

### ISSUE THREAD

**Issue by bmcdonnell** [CJ-Systems/gitflow-cjs-backup#13]
Summary: Please find an easy way to allow easy submission/migration of PRs from [petervanderdoes/gitflow-avh](https://github.com/petervanderdoes/gitflow-avh) to your repo.

I saw [your request](https://github.com/petervanderdoes/gitflow-avh/pull/403#issuecomment-1585697147):

> I'm in the process of updating gitflow. If you could kindly resubmit your pull request against my fork [here](https://github.com/CJ-Systems/gitflow-cjs) as it appears this fork is no longer being updated.

And I see that you made the same request on other open PRs there.

However, I can't create a PR on this repo from [my gitflow-avh fork](https://github.com/bmcdonnell/gitflow-avh/).

![image](https://github.com/CJ-Systems/gitflow-cjs/assets/35789100/95a5b2de-b231-4fdc-8b98-f56733ae5feb)

I could create a PR on your other repo ([CJ-Systems/gitflow-avh](https://github.com/CJ-Systems/gitflow-avh/), which was forked from [petervanderdoes/gitflow-avh](https://github.com/petervanderdoes/gitflow-avh)), but I had to manually edit the URL to start it. (GitHub was trying to make me do the PR from my branch on its upstream (petervanderdoes/gitflow-avh), and would not allow even your gitflow-avh fork in the pull-down menu (let alone your gitflow-cjs).)

![image](https://github.com/CJ-Systems/gitflow-cjs/assets/35789100/b05366f4-030c-4ecb-a427-c3fd094c2586)

When I try to do the same manual URL modification to create the PR on this repo (gitflow-cjs), it doesn't even let me.

![image](https://github.com/CJ-Systems/gitflow-cjs/assets/35789100/3544e648-4868-4844-925f-47efec5d111b)

If you want people to migrate their PRs from petervanderdoes/gitflow-avh to here, you'll probably get the most compliance if you make it as easy as possible for them to do so. I think you'll need to start by making your target repo a proper fork of petervanderdoes/gitflow-avh. Then it seems you may need to also provide instructions on how to create the PRs (which may include manually tweaking the URL, since GitHub doesn't seem to make it possible to submit a PR to a peer repo using the webUI).

**Comment by ChrisJStone** [comment]
Thank you for bringing this to my attention. I'll look into this and get it resolved and let you know.

**Comment by ChrisJStone** [comment]
[gitflow-cjs](https://github.com/CJ-Systems/gitflow-cjs)  is now setup as a direct fork of [gitflow-avh](https://github.com/petervanderdoes/gitflow-avh).  I was able to select my fork from the base repository drop down menu when I selected new pull request from your fork 
 
![image](https://github.com/CJ-Systems/gitflow-cjs-backup/assets/450713/2582fc53-f95f-4efa-af19-7d9a818c7ed4)

Since my fork of gitflow is not your upstream you would need to do this every time you wanted to submit a PR from the webUI. 

There's a ongoing discussion [Change default base repository for pull requests on forks](https://github.com/orgs/community/discussions/11729)  That deals with a similar situation. In short right now the only way to change the default base repository for PR's would be to create a new fork if using the webUI for PR's.
